### DISCUSSION TITLE
I want to split the content of a large page into multiple sections, where should I store them?

### DISCUSSION THREAD

**Discussion by galenzh** [nuxt/nuxt#25609] [Questions] [answered]
I want to split the content of a large page into multiple sections, but these sections are only used in the current page. It doesn't seem appropriate to place them in the components folder, right? However, placing them in the pages directory would automatically generate routes, which isn't what I want either. What should I do?

**Comment by TheAlexLichter** [comment, marked as answer]
You can store them in components and name them based on the page. Or you could keep them close to the page and "ignore" them when generating the routes. See @hecktarzuli's [article](https://deltener.com/blog/nuxt-enterprise-patterns-component-management/)

**Reply by galenzh** [reply]
Wow, this is exactly what I wanted. Thank you so much for your response!
